__docs__/wonder-blocks-floating/floating.stories.tsx

```diff
@@ -0,0 +1,232 @@
+import type {Meta, StoryObj} from "@storybook/react-vite";
+import {StyleSheet} from "aphrodite";
+import * as React from "react";
+
+import Button from "@khanacademy/wonder-blocks-button";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Floating} from "@khanacademy/wonder-blocks-floating";
+import Switch from "@khanacademy/wonder-blocks-switch";
+import {border, semanticColor, sizing} from "@khanacademy/wonder-blocks-tokens";
+import {BodyText} from "@khanacademy/wonder-blocks-typography";
+
+import packageConfig from "../../packages/wonder-blocks-floating/package.json";
+import ComponentInfo from "../components/component-info";
+import {longText} from "../components/text-for-testing";
+
+type StoryComponentType = StoryObj<typeof Floating>;
+
+export default {
+    title: "Packages / Floating / Floating",
```

Comment:
We can set up controls for the `placement` prop so that it's easy to tinker with in Storybook! 

<img width="995" height="158" alt="Image" src="https://github.com/user-attachments/assets/b22535f3-0599-4f76-ab4e-d2212958baf6" />